readthedocs/projects/utils.py

```diff
@@ -56,3 +57,15 @@ class Echo:
     def write(self, value):
         """Write the value by returning it, instead of storing in a buffer."""
         return value
+
+
+def get_csv_file(file_name, csv_data):
```

Comment:
I think we normally use `filename` elsewhere in the code (and in the header). Let's keep it consistent.